Practicum/Pract. 03/tasks-solutions/zad3.cpp

```diff
@@ -0,0 +1,70 @@
+#include <iostream>
+#include <fstream>
+#include <sstream>
+#define STUDENTS_MAX 1024
+#define NAME_MAX_LENGTH 16
+
+enum class HairColor {
+	BROWN,
+	BLACK,
+	BLONDE,
+	RED,
+	WHITE
+};
+
+struct Student {
+	double uspeh;
+	int facultyNumber;
+	char firstName[16];
+	char lastName[16];
```

Comment:
If the max name length is 16 => you have to allocate char array with 16 + 1, because of the terminating zero!!!

Reply:
MAX_NAME_SIZE + 1